driver.go

```diff
@@ -59,13 +59,36 @@ func (l *lvmDriver) Create(req volume.Request) volume.Response {
 		return resp(err)
 	}
 
-	cmdArgs := []string{"-n", req.Name}
+	cmdArgs := []string{"-n", req.Name, "--setactivationskip", "n"}
+	snap, ok := req.Options["snapshot"]
+	isSnapshot := ok && snap != ""
+	isThinSnap := false
+	if isSnapshot {
+		if isThinSnap, err = isThinlyProvisioned(vgName, snap); err != nil {
+			l.logger.Err(fmt.Sprintf("error checking if origin volume is thin, proceeding as if it is not: %s", err))
```

Comment:
My thought process was that an error in grep or lvdisplay doesn't necessarily mean the request arguments are invalid. Since we'll get an error from `lvcreate` if the arguments are actually invalid, we can go ahead and try to create it. Similar reasoning for the one below.

Thinking about it now, maybe I was overthinking it.